etc/tendrl/node-agent/node-agent.conf.yaml.sample

```diff
@@ -3,4 +3,6 @@ etcd_port: 2379
 etcd_connection: 0.0.0.0
 tendrl_ansible_exec_file: $HOME/.tendrl/node-agent/ansible_exec
 log_cfg_path: /etc/tendrl/node-agent/node-agent_logging.yaml
-log_level: DEBUG
\ No newline at end of file
+log_level: DEBUG
+tendrl_message_socket_addr: 127.0.0.1
```

Comment:
Needs to be a UNIX socket, as commented here: https://github.com/Tendrl/node-agent/pull/67/files#r97720923